external-builds/pytorch/pytorch_audio_repo.py

```diff
@@ -43,21 +43,11 @@
 
 THIS_MAIN_REPO_NAME = "pytorch_audio"
 THIS_DIR = Path(__file__).resolve().parent
-THIS_PATCHES_DIR = THIS_DIR / "patches" / THIS_MAIN_REPO_NAME
-
-(
-    DEFAULT_ORIGIN,
-    DEFAULT_HASHTAG,
-    DEFAULT_PATCHSET,
-    HAS_RELATED_COMMIT,
-) = repo_management.read_pytorch_rocm_pins(
-    THIS_DIR / "pytorch",
-    os="centos",
-    project="torchaudio",
-    default_origin="https://github.com/pytorch/audio.git",
-    default_hashtag="main",
-    default_patchset=None,
-)
+
+DEFAULT_ORIGIN = "https://github.com/pytorch/audio.git"
```

Comment:
for consistency if changes for `pytorch_torch_repo.py` are chosen _REPO_ should be added

Reply:
Why do you think `REPO` should be added here? `DEFAULT_ORIGIN` should be fine across the stack.